### PR TITLE
Sql scripts: Add If exists

### PR DESCRIPTION
**Pull request by ADefWebserver**
To allow the sql scripts to be run multiple times without risk of an error, each element has been wrapped in an if exists statement.

### CODE REVIEW

**Comment by sbwalker** [comment]
SQL scripts should never be executed multiple times. Can you please explain the scenario where this happened?

**Comment by ADefWebserver** [comment]
I just assumed it would happen. How would the framework know that the table (or other database object) already exists? 

For example, what if I created the table while developing the module, then installed the module using the module installer?

**Comment by sbwalker** [comment]
There is a SchemaVersions table in the database and DbUp uses this table to determine if a script needs to be executed or not. The only way a script will ever be re-executed is if the SchemaVersions table is manually modified outside of the framework. So as a result it is not necessary for a module to create its scripts in a re-executable manner.

That being said, if you are in the process of developing a module and you create the schema for it manually in the database and then run Oqtane, the system will recognize any embedded scripts in the module and will try to execute them if they have not already been executed previously. This would result in an error if the script wasnt re-executable but the error is currently being suppressed so it would not affect the framework from running.

So long story short, scripts do not need to be re-executable. However depending on your personal development approach you might want to make them re-executable.

**Comment by ADefWebserver** [comment]
> So long story short, scripts do not need to be re-executable. However depending on your personal development approach you might want to make them re-executable.

I totally understand. I also see how this particular PR is not 'really required' because it is an 'edge case' that I describe :)
